profiles/databricks.yaml

```diff
@@ -10,3 +10,5 @@ INGEST_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
 INGEST_DATA_FORMAT: spark_sql
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.75, 0.125, 0.125]
+# Specify the registry_uri if required
```

Comment:
Something like this should make more sense in terms how backend services are specified. We probably want to change how tracking URI is specified in the next iteration similarly too.
```
model_registry:
  uri: sqlite:////tmp/registry.db
```
Tracking URI in the future could be like
```
experiment_tracking:
  uri: sqlite:////tmp/tracking.db
  experiment_name: name
```